project-lifecycle.md

```diff
@@ -7,24 +7,24 @@ nav_order: 6
 
 # Project Lifecycle
 
-The term "project" within Hyperledger will refer to a
+The term "project" within Hyperledger refers to a
 collaborative endeavor to deliver a work item.
 There may be some projects that are intended to produce a document, such
 as a requirements or use cases document, a whitepaper, or analysis.
-Others may be to develop a new capability, or refactor (or remove) an
+Other projects develop a new capability, refactor, or remove an
 existing capability for the Hyperledger technology releases. Such
-projects may take the form of a new component (e.g. a new repository) or
-may propose additions, deletions or changes to an existing
-repository(s).
+projects may take the form of a new component (for example, a new repository) or
+may propose additions, deletions, or changes to an existing
+repository or repositories.
```

Comment:
suggest we adhere to conventions as  https://developers.google.com/style/plurals-parentheses?hl=en